packages/connect-core/src/utils/descriptions.ts

```diff
@@ -40,9 +40,9 @@ export async function describeTransaction(
         provider
       )
     }
-
-    // eslint-disable-next-line no-empty
-  } catch (_) {}
+  } catch (err) {
+    throw new Error(`Error: ${err}`)
```

Comment:
Should we add a bit more context here?

```suggestion
    throw new Error(`Could not describe transaction: ${err}`)
```